packages/preact/package.json

```
@@ -2,7 +2,7 @@
 	"name": "@preact/signals",
 	"version": "1.1.2",
 	"license": "MIT",
-	"description": "",
+	"description": "Manage state with style in every framework",
```

```suggestion
	"description": "Manage state with style in Preact",
```

Let's change this to 'in Preact' as this is the Preact-specific package.